compiler/main.go

```diff
@@ -28,27 +30,53 @@ var sourceFileName = map[string]string{
 }
 
 func main() {
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
+	logFile, logErr := os.OpenFile("compiler.log", os.O_RDWR|os.O_CREATE|os.O_APPEND, 0600)
+	defer logFile.Close()
+	if logErr != nil {
+		log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
+		log.Fatal().Err(logErr).Msg("Could not create log file")
+	}
+	multi := zerolog.MultiLevelWriter(consoleWriter, logFile)
+	logger := zerolog.
+		New(multi).
+		With().
+		Timestamp().
+		Str("MaRT", "compiler").
+		Logger().
+		Level(zerolog.DebugLevel)
```

Comment:
Seems to me that we could extract this code to a function, just like we did to WriteResponse on httpErrors. From my understanding, this function would only need to receive the server name ("compiler", "verdict" or "executor") and return the configured logger. 
If possible, this would remove the repetition of code, greatly improve readability of our servers and also make the development of new servers easier.

Reply:
Agreed. We actually thought about that but we were unsure of where to put the function and somehow forgot about it

Reply:
Nice! It seems this snippet would fit perfectly on the utils/logging.go file, so we could use it on the three servers we have implemented.